vaas/vaas/purger/templates/purge_form.html

```diff
@@ -1,86 +1,90 @@
-    {{ form.non_field_errors }}
-
-    <div class="control-group {% if form.url.errors %} error {% endif %}">
-        <div>
-            <div class="control-label">
-                <label class="required" for="{{ form.url.id_for_label }}">{{ form.url.label }}</label>
-
-            </div>
-            <div class="controls">
-                {{form.url}}
-                {% if form.url.errors %}
-                    <span class="help-inline">{{ form.url.errors.0 }}</span>
-                {% endif %}
-            </div>
-        </div>
-    </div>
+{% block content_title %} Purger {% endblock %}
 
-    <div class="control-group">
-        <div>
-            <div class="control-label">
-                <label class="required" for="{{ form.cluster.id_for_label }}">{{ form.cluster.label }}</label>
-            </div>
-            <div class="controls">
-                {{form.cluster}}
+{% block content %}
+    <div id="content-main" class="col-12">
+        <form action="/admin/purger/" method="post">
+            {% csrf_token %}
+            {% block form_top %}{% endblock %}
+            <div class="row">
+                {% block field_sets %}
+                    <div class="col-12 col-lg-9">
+                      <div class="card">
+                        <div class="card-body">
+                          <div class="form-group {% if form.url.errors %} error {% endif %}">
+                            <div>
+                                <div class="col-form-label">
+                                    <label class="required" for="{{ form.url.id_for_label }}">{{ form.url.label }}</label>
+                                </div>
+                                {{form.url}}
+                                {% if form.url.errors %}
+                                    <span class="help-inline">{{ form.url.errors.0 }}</span>
+                                {% endif %}
+                            </div>
+                          <div class="form-group">
+                              <div>
+                                  <div class="col-form-label">
+                                      <label class="required" for="{{ form.cluster.id_for_label }}">{{ form.cluster.label }}</label>
+                                  </div>
+                                  {{form.cluster}}
+                              </div>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                    </div>
+
+                {% endblock %}
+
+                {% block after_field_sets %}{% endblock %}
+
+                {% block inline_field_sets %}{% endblock %}
+
+                {% block after_related_objects %}{% endblock %}
+                    <div class="col-12 col-lg-3">
+                      <div class="form-actions">
+                        <div class="pull-right">
+                          <input type="submit" value="purge" class="btn btn-success form-control" />
```

Comment:
Button contents are capitalized in jazzmin, changing this for consistency
```suggestion
                          <input type="submit" value="Purge" class="btn btn-success form-control" />
```